db/migrate/20160808150745_add_metrics_status_to_ext_management_system.rb

```diff
@@ -0,0 +1,7 @@
+class AddMetricsStatusToExtManagementSystem < ActiveRecord::Migration[5.0]
+  def change
+    add_column :ext_management_systems, :last_metrics_error, :text
+    add_column :ext_management_systems, :last_metrics_update_date, :timestamp
+    add_column :ext_management_systems, :last_metrics_success_date, :timestamp
```

Comment:
:+1: for going with this solution until we have something better.
I suggest the name `last metrics_success`